as2_simulation_assets/as2_gazebo_assets/models/grayscaled_camera/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<model>
+  <name>Grayscaled Camera</name>
+  <version>1.0</version>
+  <sdf version="1.6">grayscaled_camera.sdf</sdf>
+
+  <author>
+    <name>Javi Melero</name>
+    <email>cvar-upm3@gmail.com</email>
+  </author>
+
+  <description>
+    Grayscaled camera
```

Comment:
Please @javilinos add to description more details about the camera (image size, fov, rate...)